src/RetrySettings.php

```diff
@@ -203,6 +205,8 @@ class RetrySettings
 {
     use ValidationTrait;
 
+    const DEFAULT_MAX_RETRIES = 3;
```

Comment:
Default should be `0` (unlimited retries) rather than `3`, which will change existing behavior.